src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -187,7 +188,14 @@ exchange(Peer) ->
     case mochiglobal:get(delta_mode, false) of
         true ->
             %% Anti-entropy mechanism for causal consistency of delta-CRDT.
-            gen_server:call(?MODULE, {exchange, Peer}, infinity);
+            {ok, Pid, GC_Counter} = gen_server:call(?MODULE, {exchange, Peer}, infinity),
+            MaxGCCounter = mochiglobal:get(delta_mode_max_gc_counter, 7),
```

Comment:
This default of 7 should be in an include file and documented regarding to how it works and what it's for.